tests/CompatibilityFix/MovedClassesTest.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace AmpProject\CompatibilityFix;
+
+use AmpProject\Tests\TestCase;
+
+/**
+ * Tests for AmpProject\CompatibilityFix\MovedClasses.
+ *
+ * @covers \AmpProject\CompatibilityFix\MovedClasses
+ * @package ampproject/amp-toolbox
+ */
+class MovedClassesTest extends TestCase
+{
+
+    public function testClassAliasesExist()
+    {
+        foreach (MovedClasses::ALIASES as $old => $new) {
```

Comment:
The `$new` variable is not being used:

```suggestion
        foreach (array_keys(MovedClasses::ALIASES) as $old) {
```